source/core/src/main/com/csse3200/game/components/inventory/ToolbarDisplay.java

```diff
@@ -243,4 +254,14 @@ private void updateItemSlot(int slotNum) {
             logger.info("Hotkey sound not loaded");
         }
     }
+    private void switchToolbar() {
+        window.addAction(sequence(scaleTo(1.02f, 1.02f, 0.2f, Interpolation.fade),scaleTo(1, 1, 0.2f, Interpolation.fade)));
+        try {
+            ServiceLocator.getSoundService().getEffectsMusicService().play(EffectSoundFile.SWITCH_TOOLBAR);
+        } catch (InvalidSoundFileException e) {
+            logger.error("sound not loaded");
+        }
+        curTab = (curTab + 1) % 3;
+        updateToolbar();
+    }
```

Comment:
What does this do?

This seems just to make a noise but no have any function. It seems as though it should cycle through the rows, but it doesn't do that

Reply:
Good job on the improvements, but rather than just changing what the toolbar displays, could you rather update the inventory component. This will mean the change is updated in the inventoryDisplay which will help the user understand the change.